src/envoy/http/jwt_auth/http_filter.cc

```diff
@@ -43,6 +51,12 @@ FilterHeadersStatus JwtVerificationFilter::decodeHeaders(HeaderMap& headers,
   state_ = Calling;
   stopped_ = false;
 
+  // Sanitize the JWT verification result in the HTTP headers
+  // TODO (lei-tang): when the JWT verification result is in a configurable
+  // header, need to sanitize based on the configuration.
+  SanitizeJwtVerificationResultInHeaders(
```

Comment:
Add unit test (in jwt_authenticator_test.cc) for this.

Reply:
This change is not visible to JwtAuthenticator.

Reply:
Sorry, I thought the change is in the jwt_authenticator. Ignore this then.

On a related note, there should be unit tests for http_filter. Some extra tasks to think about :)